src/System.Runtime/tests/Performance/Perf.Double.cs

```diff
@@ -12,466 +14,121 @@ namespace System.Tests
     public class Perf_Double
     {
         public static readonly double dValue = 1.23456789E+5;
-        private const int iterCount = 20000;
+        private string _string;
 
-        [Benchmark(InnerIterationCount = iterCount)]
-        [InlineData(104234.343)]
-        [InlineData(double.MinValue / 2)]
-        [InlineData(Math.PI)]
-        [InlineData(Math.E)]
-        [InlineData(double.MaxValue)]
-        [InlineData(double.MinValue)]
-        [InlineData(double.NaN)]
-        [InlineData(double.PositiveInfinity)]
-        [InlineData(double.NegativeInfinity)]
-        [InlineData(2.2250738585072009E-308)]
-        [InlineData(-2.2250738585072009E-308)]
-        [InlineData(1.1125369292536E-308)]
-        [InlineData(-0.0)]
-        [InlineData(0.0)]
-        public void DefaultToString(double number)
+        [Benchmark]
+        [InlineData(104234.343, 1_000_000)]
+        [InlineData(double.MaxValue, 100_000)]
+        [InlineData(double.MinValue, 100_000)]
+        [InlineData(double.NaN, 10_000_000)]
```

Comment:
`double.MinValue / 2` is special. It's slower than other normal numbers in current implementation. We'd better keep this one.